packages/sdk/src/MidasSdk/abi/ERC20.json

```diff
@@ -1,222 +0,0 @@
-[
```

Comment:
We actually do use this one in many places in the bots and such

Reply:
E.g. https://github.com/Midas-Protocol/monorepo/blob/fix/broken-flywheel-replace/packages/bots/liquidator/src/utils/approveTokensToSafeLiquidator.ts#L2